wpiutil/doc/datalog.ksy

```diff
@@ -0,0 +1,134 @@
+meta:
+  id: wpilog
+  title: WPILib Data Log
+  file-extension: wpilog
+  ks-version: "0.10"
+  endian: le
+  bit-endian: le
```

Comment:
I think you can also specify encoding here so you don't need to put UTF-8 everywhere.